src/components/ModiteList/styles.module.css

```diff
@@ -38,6 +38,83 @@
   height: 3rem;
 }
 
-.worldMapButton {
+.searchbarCt {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 0;
+  flex: 1;
+  align-items: center;
+  justify-content: center;
+}
+
+.searchbarWrap {
+  position: relative;
+  top: 0;
+  transform: translateY(calc(40vh / 2 - 21px));
+  transition: transform 600ms;
+  display: flex;
+  flex-direction: row;
+  z-index: 2;
+}
+
+.searchbar {
+  padding: 12px;
+}
+.searchbar input {
+  border-radius: 10px !important;
+  box-shadow: 0px 24px 48px 0px rgba(0, 0, 0, 0.5) !important;
```

Comment:
I was brought up to believe a cute, fluffy kitten dies every time someone uses `!important` in css

Reply:
I'm not sure how to fix this with Ion components in the mix.  Commence kitten death.  :(

Reply:
🐱